Doc/c-api/init.rst

```diff
@@ -1091,6 +1094,13 @@ with sub-interpreters:
    When the function returns, the current thread will hold the GIL and be able
    to call arbitrary Python code.  Failure is a fatal error.
 
+   If sub-interpreters are active, this function puts you in the global
+   interpreter (created by :c:func:`Py_Initialize`). If you create a sub-interpreter
+   right after you call this function, then the current thread is switched to that
+   sub-interpreter, and you no longer have the :c:type:`PyGILState_STATE` of the global
+   interpreter. In which case, you must call :c:func:`Py_EndInterpreter` instead of
+   :c:func:`PyGILState_Release`.
```

Comment:
I don't understand "In which case, you must call [...]". Does it mean this:

```C
/* switches to global interpreter, thread is using the global interpreter's state */
PyGILState_Ensure();
/* create a sub-interpreter */
...
/* thread is using the sub-interpreter's state */
do_something_else();
/* need to end my sub-interpreter like this ? */
Py_EndInterpreter(); 
```

Reply:
Yup. How should it be reworded?

Reply:
I don't know... I would personally wanted a rewording but I don't have the energy now to suggest one. I'm also unusure whether `you` should be used in this paragraph (that's why I reformulated the other text to remove the "you"; I don't think the "you" is used elsewhere).

Reply:
Hmm, ok. I'll see what I can come up with a little later today.

Reply:
I just pushed an update, is it more clear?

Reply:
I'll read it tomorrow. It's 2:32 AM now xD

Reply:
Dude, weren't you setting labels like an hour ago? Your dedication is impressive :stuck_out_tongue:

Reply:
Labels are easy to set! (I'm just setting labels while waiting in the lobby...)

Reply:
Yeah, I would rather be asleep than trying to deal with the issue tracker at 1:30 AM :wink:. Anyways, let me know when you get a chance to look at this.

Reply:
You'll get a review by Sunday morning (New York time)!